<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Delete</class>
 <widget class="QDialog" name="Delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image/srcc.qrc">
    <normaloff>:/img/del.jpg</normaloff>:/img/del.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qconicalgradient(cx:0.548913, cy:1, angle:0, stop:0 rgba(0, 0, 103, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLineEdit" name="product_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Product_cat">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Cat:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>-40</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Electronics</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Groceries</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Clothing</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="cat_edt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="manufact_edt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_invent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/del.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="quantity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amt_Sold:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_edt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Manufacturer">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Manufacturer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Price">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="total">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total_Qty:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_edt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="product_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Amt_edit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qty_Sold">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="qty_lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Qty_Sold:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qty_edit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="total_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prodcut_desc:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Prdct_desc_edt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image/srcc.qrc"/>
 </resources>
 <connections/>
</ui>
